governance/api.go

```diff
@@ -120,6 +120,40 @@ func (api *GovernanceKlayAPI) GetRewards(num *rpc.BlockNumber) (*reward.RewardSp
 	return reward.GetBlockReward(header, rules, pset)
 }
 
+func (api *GovernanceKlayAPI) ChainConfig() *params.ChainConfig {
+	num := rpc.LatestBlockNumber
+	return api.chainConfigAt(&num)
+}
+
+func (api *GovernanceKlayAPI) ChainConfigAt(num *rpc.BlockNumber) *params.ChainConfig {
+	return api.chainConfigAt(num)
+}
+
+func (api *GovernanceKlayAPI) chainConfigAt(num *rpc.BlockNumber) *params.ChainConfig {
```

Comment:
I think it can be factored as a private function

```
func chainConfigAt(g *Engine, num *rpc.BlockNumber)
```

Reply:
Good idea, thanks!